chapter1.md

```diff
@@ -43,55 +37,63 @@ All abundances in the compiled datasets were counts of individuals.
+
+Connolly et al. [-@connolly2014] use likelihood based methods to compare the the negative-binomial distribution (which they call the Poisson gamma) to the Poisson lognormal for a large number of marine communities. They found that they Poisson lognormal provides a substantially better fit than the negative-binomial to empirical data and that the negatvie-binomial provides a better fit to communities simulated using neutral models. They conclude that these analyses of the SAD demonstrates that marine communities are structured by non-neutral processes. Our analysis differs from that in Connolly et al. [-@connolly2014] in that they aggregate communities at larger spatial scales than those sampled and find the strongest results at large spatial scales. This may explain the difference between the two analysis or there may be differences between the terrestrial systems analyzed here and the marine systems analyzed by Connolly et al. [-@connolly2014]. The explanation for these differences is being explored elsewhere (Connolly et al. unpublished data).
+
+Alroy [-@alroy2015] compared the fits of the lognormal, log-series, Zipf, geometric series, broken stick, and a new model dubbed the "double geometric", to over 1000 terrestrial community datasets assembled from the literature. To incorporate niche partitioning models (geometric series, broken stick, and the double geometric) this research used non-standard methods for evaluating the fits of the models to the data, however the results were generally consistent those presented here. The central Kullback-Leibler divergence statistics results show that the Zipf, geometric series, and broken stick all perform consistently worse than the other distributions, and that the double geometric, log-series, and lognormal all provide the best overall fit for at least one taxonomic group (Alroy's tables S1, S2, and S3). We cannot directly assess the fits to the double geometric since it lacks a closed form probability mass function and therefore cannot be evaluated using likelihood, but in general Alroy's results our consistent with ours in that they show that the lognormal and log-series both provide better fits to the data than the Zipf. While Alroy [-@alroy2015] concludes that the data support the double geometric and not the other distributions, the Kullback-Leibler divergence results indicate that the lognormal provides equivalently good fits to the data (Alroy's tables S1, S2, and S3). Alroy's results show slightly better fits for the lognormal relative to the log-series and since the analyses did not control for differences in the number of parameters this is also consistent with our results. The double geometric is also a two parameter distribution so it is likely that if it were possible to compare its fit to that of the log-series using AICc that it would provide equivalent or slightly worse fits to the data.
```

Comment:
I like the shortening in general, but it lost a couple of details I thought were worth keeping. I'll include a hybrid version in my update that I'll push shortly.